lib/internal/Magento/Framework/Locale/TranslatedLists.php

```diff
@@ -150,10 +157,20 @@ public function getOptionCurrencies()
         $currencies = (new CurrencyBundle())->get($this->localeResolver->getLocale())['Currencies'] ?: [];
         $options = [];
         $allowed = $this->_config->getAllowedCurrencies();
+        $selectedCurrencies = explode(
```

Comment:
Maybe getting selecting countries could be done in separate protected method?

Reply:
This function will be called only once. Does it deserve it?